Doc/library/sqlite3.rst

```diff
@@ -73,27 +73,26 @@ This example uses the iterator form::
 
 .. _sqlite3-placeholders:
 
-Usually your SQL operations will need to use values from Python variables.  You
-shouldn't assemble your query using Python's string operations because doing so
-is insecure; it makes your program vulnerable to an SQL injection attack
-(see the `xkcd webcomic <https://xkcd.com/327/>`_ for a humorous example of
-what can go wrong)::
+SQL operations will usually need to use values from Python variables. It is poor
+practice to assemble queries using Python's string operations, because they are
+vulnerable to SQL injection attacks (see the `xkcd webcomic
+<https://xkcd.com/327/>`_ for a humorous example of what can go wrong)::
```

Comment:
I think addressing the reader as "you" is better here, because it is somewhat of an instruction of what _not_ to do.

```suggestion
SQL operations usually need to use values from Python variables. However, you
should beware of using Python's string operations to assemble queries, as they
are vulnerable to SQL injection attacks (see the `xkcd webcomic
<https://xkcd.com/327/>`_ for a humorous example of what can go wrong)::
```

Reply:
How about this:

```suggestion
SQL operations usually need to use values from Python variables. However,
beware of using Python's string operations to assemble queries, as they
are vulnerable to SQL injection attacks (see the `xkcd webcomic
<https://xkcd.com/327/>`_ for a humorous example of what can go wrong)::
```

Reply:
I considered that possibility. I still think it's marginally clearer as a direct instruction to the reader, but I can live with this wording :)